.goreleaser.yml

```diff
@@ -39,6 +39,29 @@ archives:
     <<: *archive_defaults
     wrap_in_directory: false
     format: zip
+brews:
+  - name: fastly
+    ids: [nix]
+    github:
+      owner: fastly
+      name: homebrew-tap
+    skip_upload: auto
+    description: Fastly CLI
+    homepage: https://github.com/fastly/cli
+    folder: Formula
+    custom_block: |
+      head do
+        url "https://github.com/fastly/cli.git"
+        depends_on "go"
```

Comment:
we also depend on a working `git` binary. unsure if we should declare it as part of the homebrew formula.

Reply:
Couple of things to clarify:
- We use [`go-git`](https://github.com/go-git/go-git) which is a pure Go implementation of git and not the local `git` binary. 
- This block is only used if the user is installing the `HEAD` and not a specific version, during which the binary is compiled via Go, its not a dependency in normal operation as we download prebuilt release assets.